index.js

```diff
@@ -11,13 +11,67 @@ if (itemNodes.length != 16) {
 
 // output flat filled matrix in console
 
-console.log(
-    itemNodes.map((item) => item.dataset.matrixId)
-)
+itemNodes[countItems-1].style.display = 'none';
+let matrix = GetMatrix(
+    itemNodes.map((item) => Number(item.dataset.matrixId))
+);
 
+setPositionItems(matrix);
 
+console.log(matrix);
 
+function GetMatrix(arr) {
 
+    const matrix = [[], [], [], []];
+    let y = 0;
+    let x = 0;
+
+    for (let i = 0; i < arr.length; i++) {
+
+    if (x >= 4) {
+        x = 0;
+        y++;
+    }
+    
+    matrix[y][x] = arr[i];
+    x++;
+}
+ 
+    return matrix;
+}
+
+
+// shuffle by clicking button
```

Comment:
it's not just shuffling, 
it handle click on shuffle button then
shufflingm then render model